packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -89,7 +89,8 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
                   <div className={this._classNames.chartTitle}>
```

Comment:
This should be fixed for StackedBarChart and MultiStackedBarChart also

Reply:
done